src/xdesign/geometry/area.py

```diff
@@ -533,6 +540,42 @@ def contains(self, other):
             return np.logical_not(border.contains_points(np.atleast_2d(x)))
 
 
+class RegularPolygon(Polygon):
+    """A regular polygon in 2D cartesian space.
+
+    It is defined by the polynomial center, order, and radius.
+
+    By default (i.e. when the ``angle`` parameter is zero), the regular
+    polygon is oriented so that one of the vertices is at coordinates
+    :math:`(x + r, x)` where :math:`x` is the x-coordinate of
+    ``center`` and :math:`r` = ``radius``. The ``angle`` parameter is
+    only meaningful modulo :math:`\pi /` ``order`` since rotation by
+    :math:`\pi /` ``order`` gives a result equivalent to no rotation.
```

Comment:
```suggestion
    only meaningful modulo :math:`2\pi /` ``order`` since rotation by
    :math:`2\pi /` ``order`` gives a result equivalent to no rotation.
```
Modulo `2PI / order`? Because there are 2PI radians in one rotation.

Reply:
Yes, good catch. Fix pushed.